CITATION.cff

```diff
@@ -26,6 +26,6 @@ authors:
 - family-names: Wong
   given-names: Susanna
 title: Kedro
-version: 0.17.6
-date-released: 2021-12-09
+version: 0.17.7
+date-released: 2022-22-02
```

Comment:
But actually this is wrong... should be YYYY-MM-DD, which is still nice because it's a palindrome.
```suggestion
date-released: 2022-02-22
```